server/index.js

```diff
@@ -37,6 +35,31 @@ app.post('/actors/add', (req, res) => {
   });
 });
 
+app.put('/actors/update', (req, res) => {
+  const {name, title, role, photo, bio, filmography, id} = req.body;
+  console.log(req.params.id);
+  console.log('PUT', name, title, role, photo, bio, filmography, id);
+  db.updateActor(name, title, role, photo, bio, filmography, id, (err, results) => {
```

Comment:
Also two different types of update may be possible. Updating Actors and updating actor information for a movie!